manageiq-operator/api/v1alpha1/manageiq_types.go

```diff
@@ -63,10 +63,6 @@ type ManageIQSpec struct {
 	// +optional
 	DatabaseVolumeCapacity string `json:"databaseVolumeCapacity,omitempty"`
 
-	// Flag to indicate if Kafka and Zookeeper should be deployed (default: false)
-	// +optional
-	DeployMessagingService *bool `json:"deployMessagingService,omitempty"`
```

Comment:
Was this ever shipped? or was this transient?  We may need to keep it since it might break backward compat.

Reply:
I see, would it better to keep the `deployMessagingService` option and set its default value to `true` as opposed to the current `false` then?